contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/index.vue

```diff
@@ -163,7 +163,8 @@
             <SpecialPermissionsList
               v-if="licenseAuditIsFinished && specialPermissions.length > 0"
               v-model="checkedSpecialPermissions"
-              :permissionIds="specialPermissions"
+              :channel-version-id="versionDetail && versionDetail.id"
+              :permission-ids="specialPermissions"
```

Comment:
Just a note that since right now, we don't return the `permissionIds` anymore in https://github.com/learningequality/studio/pull/5589/files#diff-ec56ff797d006d93cd91d7ececcc9d801eb660f39d34edebeae499ad1c64a39bR902, then we won't have this `specialPermissions` defined at this point. Now the channelVersionId will be the reference point to fetch the special permissions descriptions.

Reply:
Also note that in the useLicenseAudit.js we are still using the [getPublishedData](https://github.com/taoerman/studio/blob/a3c279887ccd15ee5c22dd53d18c99d4d4ccbdae/contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/composables/useLicenseAudit.js#L89). And this will fail because this method no longer exists in the ChannelResource.